filters/auth/basic_test.go

```diff
@@ -0,0 +1,76 @@
+package auth
+
+import (
+	"github.com/zalando/skipper/filters/filtertest"
+	"net/http"
+	"testing"
+)
+
+type createTestItem struct {
+	msg  string
+	args []interface{}
+	err  bool
+}
+
+func TestWithMissingAuth(t *testing.T) {
+	spec := NewBasicAuth()
+	f, err := spec.CreateFilter([]interface{}{"htpasswd.test"})
+	if err != nil {
+		t.Error(err)
+	}
+
+	req, err := http.NewRequest("GET", "https://www.example.org/", nil)
+	if err != nil {
+		t.Error(err)
+	}
+
+	expectedBasicAuthHeaderValue := ForceBasicAuthHeaderValue + "\"Basic Realm\""
```

Comment:
Small note, you can use the back tick to include the double quote as part of the string, like
```
`"Basic Realm"`
```